tsl/src/compression/create.c

```diff
@@ -245,7 +245,7 @@ compresscolinfo_init(CompressColInfo *cc, Oid srctbl_relid, List *segmentby_cols
 		{
 			ereport(ERROR,
 					(errcode(ERRCODE_SYNTAX_ERROR),
-					 errmsg("column %s in compress_segmentby list does not exist",
+					 errmsg("column %s in timescaledb.compress_segmentby list does not exist",
```

Comment:
```suggestion
					 errmsg("column \"%s\" specified in option timescaledb.compress_segmentby does not exist",
```

I believe it is standard practice to double quote identifiers like columns in PostgreSQL error messages. 

Here's an example from PostgreSQL:

`"column \"%s\" of relation \"%s\" does not exist"`